comet/utils/memory.py

```
@@ -0,0 +1,140 @@
+    except Exception:
+        return False
+
+
+def _trim_with_libc() -> bool:
+    if sys.platform == "linux":
+        malloc_trim = _get_linux_malloc_trim()
+        if malloc_trim is None:
+            return False
+        try:
+            return bool(malloc_trim(0))
+        except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)